app/components/roadmap-info-alert.ts

```diff
@@ -0,0 +1,20 @@
+import Component from '@glimmer/component';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    heading: string;
+    class?: string;
```

Comment:
You don't need to pass in `class` like this in ember. Look at how we handle other components